### PR TITLE
Groovyw

### PR DESCRIPTION
**Pull request by Steampunkery**
Adds a groovyw script for managing modules similar to Terasology's.

### CODE REVIEW

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Thu Dec 24 21:00:34 CAT 2015
+#Fri Jan 27 18:22:28 EST 2017
```
Comment (vampcat) [inline gradle/wrapper/gradle-wrapper.properties]: Do we need this line?

Comment (Steampunkery) [reply gradle/wrapper/gradle-wrapper.properties]: It's a copy/paste. I'll remove it. It was probably auto generated by whatever gradle script made what I copied.

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Pull request merged**